src/adr/articles/0008-integration-with-sirius.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: layouts/adr.njk
+title: "Integration with Sirius"
```

Comment:
I've had a think about this and I think what tripped me up is the framing as "integration with Sirius", which is not really what we're doing. We're integrating with the OPG's data that Sirius happens to be the default window on and current owner of the data schema. Both of which are things that can and will change over time.

Reply:
Yeah I agree, as I started writing it became really clear this wasn't the actual thing we wanted to do and is confusing. I think I cover this in the article however the title now doesn't match this. Do you think changing the title to `Integration with OPG data` is better and adjust content where needed to match that title?

Reply:
I think the reframing might help. I wonder if really it's about the end to end visibility of data for business processes.

Is the decision really, Modernise and Sirius both need to access the same data and for it to be a single source of truth? Is that source a new LPA Data service that sits initially as part of Sirius but is eventually independent of the two?

Reply:
Have adjusted the content to what I think better reflects our current setup and context